set_env.sh

```
@@ -0,0 +1,28 @@
+  echo "[set_env] Missing $ENV_FILE  (copy from .env.template)" >&2
+  exit 1
+fi
+
+# ── Reliable exporter ──
+set -a          # automatically export every variable that gets set
+. "$ENV_FILE"   # 'source' preserves order, so ${VAR} references work
+set +a
+# ──────────────────────
+
+# Jump to repo root so `bun run dev` finds the root package.json
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)